run_configs/test_all.sh

```diff
@@ -0,0 +1,12 @@
+set -e
+
+cd ~/git/fab/run_configs/gcom
+python -m grab_gcom
+python -m build_gcom_ar
+python -m build_gcom_so
+
+cd ../jules
+python -m build_jules
+
+cd ../um
+python -m build_um
```

Comment:
I notice that Gung Ho is not part of "all." There appears to be a change ordering issue here.

Reply:
Yep, that's in a separate PR and we'll be adding it here when it's merged, sooner or later.